### ISSUE TITLE
Remove "Build Unity Project" console warnings

### ISSUE THREAD

**Issue by ChristophHoltmann** [Microsoft/MixedRealityToolkit-Unity#2364]
## Overview

Remove unnecessary CS0649 warnings in the console window when building a Unity project by clicking the "Build Unity Project" button in the MRTK "Build Window"

## Expected Behavior

No CS04649 warnings should appear in the console window

## Actual Behavior

CS04649 warnings from the MRTK appear in the console window

## Steps to reproduce

1. Build a Unity project by clicking the "Build Unity Project" button in the "Build Window"
2. The CS0649 warnings appear in the console window

## Mixed Reality Toolkit Release Version

Master

**Comment by StephenHodgson** [comment]
Pretty sure we just closed an issue about this.

@keveleigh

**Comment by keveleigh** [comment]
Yep! This is fixed in the June branch via #2224 and #2360 (and the other issue referenced is #2068). We'll make sure to be more diligent in removing these warnings before putting out a release in the future.
